Automattic/wp-calypso
# Store: Refactor getQueries to reuse in other parts of store stats

Per the feedback [here](https://github.com/Automattic/wp-calypso/pull/23344/files/aea4c294a06bba130e63ec5f21c0633e5da6fbbd#r174933961), getQueries has been moved to `store-stats` and updated to be more reusable with the rest of the stats code. It accepts a unit and base date. You can also pass overrides if you need to overwrite things like the default quantity. cc @psealock does this work for you?

To Test:
* Load up your dashboard and make sure data still loads and displays properly in the stats widget
